### PR TITLE
Datajam changes into master

### PR DESCRIPTION
**Pull request by ebiiii**
Let's use this pull request to review the changes (it's easier this way to see the actual diff)

### CODE REVIEW

File: src/components/tutorial/index.ts [hunk 1]
```diff
@@ -222,7 +209,7 @@ export class TutorialComponent extends Vue {
                 if (response.status === 201) {
                     let payload = JSON.stringify({
                         name: this.project_name,
-                        backend: 'local',
+                        backend: 's3',
```
Comment (ebiiii) [inline src/components/tutorial/index.ts]: let's parametrize the default backends

File: src/components/tutorial/index.ts [hunk 2]
```diff
@@ -328,63 +319,63 @@ export class TutorialComponent extends Vue {
...
-                }
-            ).then(response2 => {
-                return response2.json()
-                }
-            ).then(response2 => {
-                console.log('executing', response2)
-                this.progress = false
-                router.push(`/deploy/context/${response.identifier}/execution/${response2.identifier}`)
+        if (this.nbIDE > 0) {
+            duplicateFile(this.nbIDE, 0, 'Notebook copy.ipynb', this.project.id).then(r => {
```
Comment (ebiiii) [inline src/components/tutorial/index.ts]: make the copy optional with a checkbox

File: src/components/tutorial/index.ts [hunk 3]
```diff
@@ -328,63 +319,63 @@ export class TutorialComponent extends Vue {
...
-        this.do_execution(this.dockers()[this.imageIDE], '')
+    do_executeIDE() {
 
+        createContext(this.imageIDE, ['8888'], [],
+            this.project.id, this.nbIDE
+        ).then(response => {
+            console.log('create', response)
+            return response.json()
+        }).then(responsej => {
+            runContext('k8s', '', responsej.identifier).then(r => {
```
Comment (ebiiii) [inline src/components/tutorial/index.ts]: parametrize backend

File: src/components/tutorial/tutorial.html [hunk 4]
```diff
@@ -43,23 +30,22 @@ <h3 class="headline mb-0">Choose an existing one</h3>
...
                 <v-spacer></v-spacer>
-                <v-btn secondary @click="chooseProject()">Next</v-btn>
+                <v-btn secondary @click="chooseProject()">Select and continue</v-btn>
               </v-card-actions>
             </v-card>
           </v-flex>
         </v-layout>
       </v-stepper-content>
       <v-stepper-content :step="2">
-        <div>
```
Comment (ebiiii) [inline src/components/tutorial/tutorial.html]: Should we keep the "import" option or not ?

File: src/components/user/index.ts [hunk 5]
```diff
@@ -116,7 +116,7 @@ export class UserMenuComponent extends Vue {
             if (state.logged_in) {
                 this.items.push({ title: 'Execution contexts', icon: 'settings_applications', path: '/deploy/context' })
                 this.items.push({ title: 'Storage', icon: 'folder', path: '/storage' })
-                this.items.push({ title: 'Graph', icon: 'share', path: '/graph' })
+//                this.items.push({ title: 'Graph', icon: 'share', path: '/graph' })
```
Comment (ebiiii) [inline src/components/user/index.ts]: do we keep it for the moment?

File: src/main.ts [hunk 6]
```diff
@@ -34,6 +34,15 @@ import { GraphItemListComponent, GraphItemTableComponent } from './components/gr
 import { BucketDialogComponent, ContextDialogComponent, ExecutionDialogComponent,
     ProjectDialogComponent, VersionDialogComponent } from './components/dialogs'
 
+// Activate sentry if a DSN is provided through the environment variables
+import Raven from 'raven-js';
+import RavenVue from 'raven-js/plugins/vue.js';
+
+Raven
+    .config('https://250bfa79136d4bc2b19d4f8d977b4f5f@sentry.io/246730')
```
Comment (ebiiii) [inline src/main.ts]: secret url ?

Comment (ableuler) [reply src/main.ts]: I think the goal here should be to respect the SENTRY_DSN environment variable like all the other services. Will be a bit more tricky than for the backend stuff, but I'm trying to find a not too hacky way to do this.

File: src/components/user/index.ts (hunk 5 above)
Comment (ableuler) [reply src/components/user/index.ts]: Should we leave it out and retry with the graph as soon as the graph view displays only the current project?

File: src/components/tutorial/tutorial.html (hunk 4 above)
Comment (ableuler) [reply src/components/tutorial/tutorial.html]: Having the collaborative aspects of renga in mind, I think it's actually a nice feature. But it means that I will have to loosen on the assumption that every file is in a bucket which belongs to the project.

File: src/components/tutorial/index.ts (hunk 2 above)
Comment (ableuler) [reply src/components/tutorial/index.ts]: 👍

File: src/components/tutorial/index.ts (hunk 3 above)
Comment (ableuler) [reply src/components/tutorial/index.ts]: For the storage we can do a query to get the available backends. The same would be nice for the deployer.

File: src/components/tutorial/index.ts (hunk 1 above)
Comment (ableuler) [reply src/components/tutorial/index.ts]: ok

**Pull request merged**